Detectors/TPC/reconstruction/macro/findKrBoxCluster.C

```diff
@@ -58,44 +58,28 @@ void findKrBoxCluster(int lastTimeBin = 1000, int run = -1, int time = -1, std::
     clFinder->loadGainMapFromFile(gainMapFile);
   }
 
+  // clFinder->setMinNumberOfNeighbours(0);
+  // clFinder->setMinQTreshold(0);
+  clFinder->setMaxTimes(lastTimeBin);
+
   // Now everything can get processed
   // Loop over all events
   for (int iEvent = 0; iEvent < nEntries; ++iEvent) {
     std::cout << iEvent + 1 << "/" << nEntries << std::endl;
     tree->GetEntry(iEvent);
-    // Each event consists of sectors (atm only two)
 
     for (int i = 0; i < 36; i++) {
       auto sector = digitizedSignal[i];
       if (sector->size() == 0) {
         continue;
       }
+      clFinder->setSector(i);
+      clusters = clFinder->loopOverSector(*sector);

```

Comment:
```suggestion
      clFinder->loopOverSector(*sector);
```
With my other comments.